compiler/z/codegen/ConstantDataSnippet.cpp

```diff
@@ -406,71 +406,81 @@ uint8_t *
 TR::S390InterfaceCallDataSnippet::emitSnippetBody()
    {
 
-//  64-Bit Layout.
-//  0x00000100211cfe58     DC  0x100211cf58a    ; Call Site RA
-//  0x00000100211cfe60     DC  0x8020b380       ; Address Of Constant Pool
-//  0x00000100211cfe68     DC  0x00000040       ; CP Index
-//  0x00000100211cfe70     DC  0x00000000       ; Interface class
-//  0x00000100211cfe78     DC  0x00000000       ; Method Index
+//  64-Bit Layout
+//  Note: some fields and PIC slots will be unused if interface call turns out to be a direct dispatch.
 //
-//  0x00000100211cfe80     DC  0x00000000       ; Flags (resolved)
-//  0x00000100211cfe88     DC  0x100211cfe90    ; lastCachedSlotField
+//  DC  0x100211cf58a    ; Call Site RA
+//  DC  0x8020b380       ; Address Of Constant Pool
+//  DC  0x00000040       ; CP Index
+//  DC  0x00000000       ; Interface class
+//  DC  0x00000000       ; Method Index    (contains Method ptr if the target is direct)
+//  DC  0x00000000       ; Thunk address   (used if the target is direct dispatch)
+//  DC  0x00000000       ; Flags (resolved=1, unresolved = 0)
+//  DC  0x100211cfe90    ; lastCachedSlotField
```

Comment:
Can we pad the 64-bit fields with zeros to signify to the reader that they are 64-bit. For example currently it is non-obvious that "Address Of Constant Pool" is a 64-bit address.